apps/wallet/src/ui/app/hooks/useGetTransactionsByAddress.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { type SuiTransactionResponse, type SuiAddress } from '@mysten/sui.js';
+import { useQuery } from '@tanstack/react-query';
+
+import { useRpc } from '_hooks';
+
+export function useGetTransactionsByAddress(address: SuiAddress | null) {
+    const rpc = useRpc();
+
+    const response = useQuery<SuiTransactionResponse[], Error>(
+        ['transactions-by-address', address],
+        async () => {
+            if (!address) return [];
+            const txnIdDs = await rpc.getTransactions({
+                ToAddress: address,
```

Comment:
I think we used to request from also `fromAddress` transactions, did we remove it for some reason?